models/airline.go

```diff
@@ -1,7 +1,17 @@
 package models
+
 type Airline struct {
 	Id    string `json:"id"   gorm:"primaryKey;autoIncrement"`
 	Name  string `json:"name" binding:"required" gorm:"unique;notNull;size:100"`
 	Count int    `json:"count" gorm:"null"`
 }
 
+func (airline *Airline) SetName(name string) Airline {
+	(*airline).Name = name
+	return *airline
```

Comment:
Is there an idiomatic way to collapse these 2 lines into a single line?

Reply:
Searched for it, we cannot do it in idiomatic way.